layouts/partials/sidebar.html

```diff
@@ -14,15 +14,15 @@
 		<div role="button" class="showDropdown bg-near-white h3 w3 flex items-center justify-center" tabindex="0">
 			<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-menu"><line x1="3" y1="12" x2="21" y2="12"></line><line x1="3" y1="6" x2="21" y2="6"></line><line x1="3" y1="18" x2="21" y2="18"></line></svg>
 		</div>
-		<div class="dn db-ns dropdown overflow-y-hidden pv4-ns pv3">
+		<div class="dn db-ns dropdown overflow-y-auto pbv4-ns mt4-ns">
```

Comment:
replaced padding-top with margin-top to improve spacing when the section is scrolled